src/cascadia/ut_app/TerminalApp.Unit.Tests.manifest

```diff
@@ -9,6 +9,7 @@
     <application>
         <!-- Windows 10 1903 -->
         <!-- See https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/xaml-islands -->
+        <!-- "maxversiontested" is CASE SENSITIVE. Do not change this.-->

```

Comment:
this keeps flip-flopping on certain changes. we should do a pass to make sure only source files change for each SA commit.

Reply:
I really don't know why. It just seems to flip of its own accord when I build the entire solution. I intend to ensure that I make it match master before merging.